### PR TITLE
As add scala-fixture to build definition

### PR DESCRIPTION
**Pull request by amory-coursera**
Built locally, worked properly. @bryan-coursera ptal.

### CODE REVIEW

File: project/Build.scala [hunk 1]
```diff
@@ -222,7 +222,9 @@ object Courier extends Build with OverridablePublishSettings {
     s";++$currentScalaVersion;project scala-generator;$publishCommand" +
     s";++$sbtScalaVersion;project sbt-plugin;$publishCommand" +
     s";++$sbtScalaVersion;project scala-runtime;$publishCommand" +
-    s";++$currentScalaVersion;project scala-runtime;$publishCommand"
+    s";++$currentScalaVersion;project scala-runtime;$publishCommand" +
+    s";++$sbtScalaVersion;project scala-fixture;$publishCommand" +
```
Comment (bryankane) [inline project/Build.scala:226]: since this isn't an sbt plugin, i don't know if we need this line

Comment (amory-coursera) [reply project/Build.scala:226]: I was pattern matching against the `scala-runtime` steps here. If you're confident this isn't needed, I'm happy to remove.

Comment (bryankane) [reply project/Build.scala:226]: i'm most definitely not positive, but _pretty_ sure.

Comment (amory-coursera) [reply project/Build.scala:226]: Discussed offline. We'll keep artifacts for both scala versions, because they might be necessary and it doesn't hurt.

**Pull request merged**